pkg/deps/project.json

```diff
@@ -1,6 +1,6 @@
 {
   "dependencies": {
-    "Microsoft.NETCore.Platforms": "1.0.1"
+    "Microsoft.NETCore.Platforms": "1.0.2-preview1-24513-01"
```

Comment:
Looks like the preview1 versions are still 1.0.2. The ones in master look like they were bumped to 4.3.0 with all the other packages. @ericstj does that sound right?

For the purpose of this repo it is only using this to get the runtimes to validate against so it actually doesn't matter much. I think we could probably even eliminate it completely now as BuildTools has a default version nowadays.